soh/soh/UIWidgets.cpp

```diff
@@ -227,7 +227,7 @@ namespace UIWidgets {
         }
     }
 
-    void EnhancementCombobox(const char* name, const char* ComboArray[], size_t arraySize, uint8_t FirstTimeValue) {
+    bool EnhancementCombobox(const char* name, const char* ComboArray[], size_t arraySize, uint8_t FirstTimeValue) {
```

Comment:
You'll still need to make the change to the method definition to actually return the bool here, you can reference `EnhancementCheckbox` for an example